src/components/ExploreOtherTools/index.js

```diff
@@ -38,7 +38,7 @@ const ExploreOtherTools = ({ title, items, ...props }) => {
             ))}
           </Carousel>
         </Hidden>
-        <Hidden mdDown implementation="css">
+        <Hidden lgDown implementation="css">
```

Comment:
Why are we going md to lg?

Reply:
@kilemensi I just noticed also this was generated by the codemod. When we revert it back, the site's responsiveness breaks.

Reply:
Why @kelvinkipruto is what I'd like to know.